plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -1936,6 +1936,76 @@ public void testCreateNestedPartitionedTable()
         dropTable("test_nested_table_3");
     }
 
+    @Test
+    public void testCreateWithNestedPartitionedColTable()
+    {
+        @Language("SQL") String createTable = "" +
+                "CREATE TABLE test_nested_partition_table_1 (" +
+                " bool BOOLEAN" +
+                ", int INTEGER" +
+                ", arr ARRAY(VARCHAR)" +
+                ", big BIGINT" +
+                ", rl REAL" +
+                ", dbl DOUBLE" +
+                ", mp MAP(INTEGER, VARCHAR)" +
+                ", dec DECIMAL(5,2)" +
+                ", vc VARCHAR" +
+                ", vb VARBINARY" +
+                ", ts TIMESTAMP(6)" +
```

Comment:
i would assume that in this test the partition field is all that matters; do we need to have all these data columns?